ahnlich/typegen/src/main.rs

```diff
@@ -61,8 +60,6 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .to_owned()
                 .unwrap_or(std::path::PathBuf::from(SDK_PATH));
 
-            std::fs::create_dir_all(output_dir).expect("Failed to create sdk directory");
```

Comment:
Why was this

Reply:
Yeah, The idea was you provide an output dir, or use the default one.
It opens the door to other considerations such as permission issues.(Imagine the output dir in /etc/)